app/controllers/games_controller.rb

```diff
@@ -21,6 +21,7 @@ def create
   end
 
   def show
+    @game = current_game
```

Comment:
This line could be removed and everything would still work. Maybe we should call current_game in the view for consistency?